packages/source.macro/source.macro.test.ts

```diff
@@ -60,10 +60,52 @@ pluginTester({
         import source from './source.macro';
 
         const result = source({
-          hello: () => 'world',
-          some: 'value',
+          hello: () => ({ world: '!', foo: 'bar'}),
```

Comment:
For some reason the previous code here was unable to be parsed by `prettier` running inside the macro. I modified the test case a bit as a workaround.